datalake/processing_stack.py

```diff
@@ -21,12 +22,32 @@
 class ProcessingStack(core.Stack):
     """Data Lake processing stack definition."""
 
-    # pylint: disable=too-many-locals
+    # pylint: disable=too-many-locals,too-complex
```

Comment:
Can we simplify this code to avoid this complexity? For example, there are at least three places where we check `if deploy_env == "prod"` - those can all be joined into a single `if`/`else` to set some configuration.

Reply:
Should be fixed in 17923d87546f6713a687ded8ccb111eff35e4b34